tensorboard/components/tf_ng_tensorboard/polymer_interop/reloader/reloader.component.ts

```diff
@@ -21,13 +21,13 @@ import {
   State,
   getReloadEnabled,
   getReloadPeriodInMs,
-} from '../core/core.reducers';
-import {reload} from '../core/core.actions';
+} from '../../core/core.reducers';
+import {reload} from '../../core/core.actions';
 
 /** @typehack */ import * as _typeHackRxjs from 'rxjs';
 
 @Component({
-  selector: 'reloader',
+  selector: 'reloader-interop',
```

Comment:
Could you please explain why is this renamed to "*-interop"?  I thought any file under "-interop" sounds like it should be safe to remove one day, once we get rid of Polymer.  That applies to storage-interop, but not reloader-interop.

Reply:
You are right. We would have this component without Polymer. I reverted the change back. Sorry!